src/__tests__/lockfile.test.ts

```diff
@@ -1,20 +1,35 @@
 import { setupFixture } from "../helpers/setupFixture";
 import { parseLockFile } from "../lockfile";
 
+const ERROR_MESSAGES = {
+  NO_LOCK: "You do not have yarn.lock, pnpm-lock.yaml or package-lock.json. Please use one of these package managers.",
+  UNSUPPORTED: "Your package-lock.json version is not supported: 1. You need npm v7 or above and package-lock v2 or above. Please, upgrade npm or choose a different package manager.",
```

Comment:
The `${npmLock.lockfileVersion}`. I rewrote the error message to be more specific.

Reply:
It's about the version of the lockfile format in package-lock.json, right? I think that's fine. Maybe the wording can be fixed up a bit, but it would make sense to the folks using npm for workspaces.